features/install-profiles/src/main/feature/feature.xml

```diff
@@ -48,7 +48,6 @@
              description="This will install all the applications for development purposes."
              start-level="3">
         <feature>profile-standard</feature>
```

Comment:
Based on what we have left in this profile, looks like a good candidate to remove